hooks-admin.php

```diff
@@ -199,6 +199,9 @@
 	add_filter( 'attachment_fields_to_edit', '\Pressbooks\Admin\Attachments\add_metadata_attachment', 10, 2 );
 	add_filter( 'attachment_fields_to_save', '\Pressbooks\Admin\Attachments\save_metadata_attachment', 10, 2 );
 	add_filter( 'edit_form_top', [ '\Pressbooks\Book', 'notifyBisacCodesRemoved' ] );
+	add_action( 'save_post', [ 'Pressbooks\DataCollector\User', 'storeLastActiveDate' ], 10 );
+	add_action( 'edit_term', [ 'Pressbooks\DataCollector\User', 'storeLastActiveDate' ], 10 );
+	add_action( 'switch_theme', [ 'Pressbooks\DataCollector\User', 'storeLastActiveDate' ], 10 );
```

Comment:
I think it could be, dunno if it could be dependent on the user class instance. I think this instance is created in any case, so we could move it here. Do you think it should?

Reply:
I think it would be better, yes. I mean, all hooks related to user data collector will be in one place. And yes, the instance is created [here](https://github.com/pressbooks/pressbooks/blob/d3d86176710877a939be35de73630437da7c7e0f/hooks.php#L65), so it should be fine.